### PR TITLE
feat(desktop): add startup receipts and service-owner stop gate

### PR DESCRIPTION
**Pull request by maxvint**
## Capability and frozen contract

Shapes frozen; implementation and focused validation complete.

- `vibe start` and `vibe start --no-open-browser` emit one exact-prefix schema-v1 startup receipt, preserving the human-readable output.
- Process identity and reuse are captured inside `start_service` / `start_ui`. Service provenance, not a caller-side pre-launch snapshot, selects `outcome`; scoped launches update identity to the authoritative service PID.
- `vibe stop --receipt <json>` checks the recorded service PID and live psutil create-time (inclusive ±2 ms) before any shutdown side effect. Mismatch or malformed input returns exit 3 and reason JSON; a match follows the existing full graceful stop, including its exit-2 failure path. Unscoped stop is unchanged.
- Frozen producer/consumer specification: `docs/plans/desktop-start-receipt.md`.

## Dependencies and integration

- Base: `desktop`, including merged #1972 (`0fe5ce480`). No unmerged implementation dependency or stacked branch.
- This is the producer contract consumed by the independent Rust tray lifecycle lane #1975. The consumer must cite this PR as its integration dependency.
- No changes to `desktop/**`, workflows, deployment, or the consumer-owned lifecycle plan.

## Evidence layers

- Unit / producer contract: 368 tests pass across CLI, runtime service locks, scoped bootstrap, desktop Runtime helpers, restart supervisor, and state migration guards, and legacy setup-status callers.
- Coverage includes actual startup functions with fake process creation, every service reuse-return path, stale caller snapshots, service/UI launch-captured psutil timestamps, scoped and late authoritative-PID adoption, both browser modes, strict malformed-receipt rejection, inclusive tolerance, PID replacement/recycling, no-side-effect refusal, legacy full-stop behavior, and parser-to-dispatch propagation.
- Static checks: Ruff passes for all changed Python files; `git diff --check` passes.
- Scenario catalog: no dedicated startup-receipt scenario ID exists; coverage extends the existing `tests/test_vibe_cli.py` suite.
- Residual manual / integration: the orchestrator's desktop acceptance pass must exercise real shell startup, adoption, and receipt-conditioned stop across Rust/Python. The local agent-hosting service was not restarted; no production or regression product state was modified.

## Known-by-design ledger

- **Service-owner gate, not per-process capability:** after the service identity matches, today's service sweep, UI, OpenCode, and tunnel shutdown remain intentionally unchanged. This v1 gate does not prevent same-install concurrent replacement after gate time, nor independently authorize each UI or helper process. A stronger guarantee would require a new contract version.
- **Ownership policy belongs to the consumer:** `outcome` describes the service's actual spawn/reuse branch only; UI reuse does not change it. Python accepts `outcome=reused` as identity input. The desktop consumer treats a `reused` receipt as adoption regardless of which process invoked start, never offers stop for an adopted runtime, and must never retry a refused scoped stop as unscoped stop.
- **No secret or same-user isolation claim:** the receipt contains process identity, not a capability token; existing same-user unscoped stop remains available.

- **English-only CLI flag help (explicit scope ruling):** the task brief explicitly requires English-only strings in this CLI path, with no new i18n keys. `VibeArgumentParser` extends parsing and error reporting, not `format_help`; the existing parser help strings, including adjacent start/stop options, are English and do not load the configured locale. An isolated render with a Chinese configuration confirmed English start/stop help and zero configuration-language reads. Localizing only `--receipt` would create a lone inconsistent translated flag. A future localization effort must address the CLI help surface consistently as a separate cross-cutting change; it is outside this receipt contract. The orchestrator reaffirmed this non-change after review comment 3975755196.

Do not merge automatically; exact-head Codex review, zero unresolved threads, and all expected CI checks remain delivery gates.

### CODE REVIEW

File: vibe/cli.py [hunk 1]
```diff
@@ -13897,7 +13942,11 @@ def build_parser():
     parser = VibeArgumentParser(prog="vibe")
     subparsers = parser.add_subparsers(dest="command")
 
-    subparsers.add_parser("stop", help="Stop all services")
+    stop_parser = subparsers.add_parser("stop", help="Stop all services")
+    stop_parser.add_argument(
+        "--receipt",
+        help="Stop only if the service identity matches this startup receipt JSON.",
```
[the remark this answers is not in the document]
Comment (maxvint) [reply vibe/cli.py:13948]: Known-by-design, confirmed by the orchestrator after reviewing this finding. The task brief explicitly requires English-only strings in this CLI path with no new i18n keys. The existing `VibeArgumentParser` extends parsing and error reporting only; it does not override `format_help`, and flag-help construction does not load the configured locale. The adjacent start/stop help and the rest of this parser surface are English. I also rendered both help pages in isolated test-owned state with a Chinese configuration: both remained English, with zero configuration-language reads.

Localizing only this option would create an inconsistent one-off rather than preserve a currently localized flag-help surface. A complete CLI help localization pass is a separate cross-cutting change. The PR body now records this scope decision under **Known-by-design ledger → English-only CLI flag help (explicit scope ruling)**. No product code or validation is being changed for this finding; resolving under that explicit ruling and requesting a fresh review of the same head.
